odoo/tools/translate.py

```diff
@@ -177,6 +177,12 @@ def translate_attrib_value(node):
 
 avoid_pattern = re.compile(r"\s*<!DOCTYPE", re.IGNORECASE | re.MULTILINE | re.UNICODE)
 
+def force_translatable(node):
+    # Some nodes have an auto-updated DOM structure (e.g. text highlight
+    # effects) that makes them impossible to translate, the goal here is
+    # to allow them to be translated as a whole.
+    classes = node.attrib.get("class", "").split(" ")
+    return any(allowed_class in classes for allowed_class in ["o_text_highlight", "o_text_highlight_item"])
```

Comment:
@qsm-odoo @xmo-odoo since the text highlights are saved in a simplified format (without inline SVGs), the translation issue is automatically solved, and we don't need this adaptation code anymore :+1: